jsonnet/rules.libsonnet

```
@@ -320,6 +320,10 @@ local droppedKsmLabels = 'endpoint, instance, job, pod, service';
             record: 'cluster:control_plane:all_nodes_ready',
             // Returns 1 if all control plane nodes are ready and is absent otherwise. Should be used to suppress alerts during control plane upgrades or disruption.
           },
+          {
+            expr: 'count(alertmanager_receivers)',
```

I'd rather look at the max value in the openshift-monitoring namespace.

```suggestion
            expr: 'max(alertmanager_receivers{namespace="openshift-monitoring"})',
```

---

SGTM, fixed.